### ISSUE TITLE
For SCC calculation: check mitigation and adaptation links

### ISSUE THREAD

**Issue by jkikstra** [openmodels/mimi-page-2020.jl#30]
@P-Wlich @jkikstra 
In the current scc_testversion branch, the contribution of adaptation and mitigation costs to the SCC values is zero.
a) Check if this should be the case by more closely exploring the links between RCP/SSP scenarios, (and older PAGE Policy A/B) with how adaptation and mitigation costs are calculated.
b) Explain whether or not this makes sense theoretically when simulating a scenario.

**Comment by pwaidelich** [comment]
For adaptation:
a) The adaptation component (AdaptationCosts.jl, TotalAdaptationCosts.jl) takes only GDP and population as inputs from other components. There is no explicit reaction to changes in climatic outcomes and hence the pulse does not affect adaptation efforts or costs. This is confirmed for SCC by Hope (2013: Critical issues for the calculation of the social cost of CO2, p. 537): "But adaptation costs do not affect the SCCO2 because the amount of adaptation would not be expected to vary with small changes in emissions" (although the exogenously set adaptation efforts obviously drive SCC outputs).
b) I think it is plausible because SCC gives the damages of a marginal tonne of emissions holding everything else equal. SCC values are meant to inform policy in combination with information about MAC estimates, so endogenously adjusted adaptation policy might undermine this purpose to some degree. As previous SCC estimates based on PAGE have not taken changed adaptation efforts into account, we would also face comparability issues and the challenge to change yet another part of the model just for our theses.

Long story short, I would leave it the way it is :)

**Comment by pwaidelich** [comment]
For mitigation:
a) The mitigation component (AbatementCosts.jl, AbatementCostParameters.jl, TotalAbatementCosts.jl) uses two climatic inputs for each GHG, e0_baseline[GHGname]emissions which is determined by a csv file in /data/ and er_[GHGname]emissionsgrowth calculated in RCPSSPScenario.jl. As our carbon pulse shocks only e_globalCO2emissions in CO2emissions.jl, this leads to a zero impact on mitigation costs. In theory, it would be feasible to adjust er_CO2emissionsgrowth for the mitigation component, i.e. to increase it in the first timestep and lower it in the second according to the CO2 pulse.
Hope (2013, p. 539) sets different impact sectors for PAGE (market damages, non-market, SLR, discontinuity) to zero and evaluates how the SCC drops. As he argues that "it might be thought that the four drops in Table 2 should sum to 100 %; however they do not sum exactly as there are small interactions between them", changes in mitigation costs did not feed into PAGE09 SCC estimates.
b) To be discussed

Preliminary conclusion: We could leave it the way it is and should certainly compute SCC estimates with mitigation cost held constant for comparability with previous estimates. Not sure if it's the best way to go, though.
@fhdezram Did you include changes in mitigation costs into your SCC estimates?

**Comment by pwaidelich** [comment]
Closing this as SCC has been implemented into the master branch
